README.md

```diff
@@ -92,6 +92,22 @@ Other operating systems, such as Microsoft Windows, which access files in a case
 
 Best practice is to use `data-e2e="your_test_id"` element attributes to select specific elements, eg `cy.get('[data-e2e="cta-button-1"]').click()` instead of `cy.get('.container .infobox a.button').contains('DOWNLOAD').click()`.
 
+|                       | Included in Cypress Test Production?     |  What is it used for?                                                                |
+|-----------------------|-|-------------------------------------------------------------------------------------------------------------------------------|
+| check_links.js        |✘ (takes a long time to finish running)   | Detect broken links throughout the website. Result is saved in `logs` folder         |
```

Comment:
You don't have to write this, but the reason that `check_links.js` is not included in the production test is because it does not reliably pass. It depends on external website over which we have no control.